### PR TITLE
Fixed permission handling on browser:menuItem

### LINKED ISSUE #26: permissions ignored
it looks like [getMenuItemByAction](https://github.com/plone/plone.app.contentmenu/blob/master/plone/app/contentmenu/displayviewsmenu.py#L18-L26) ignores permissions.

so you can't distinct view_methods based on permission.

### PR DESCRIPTION
**Pull request by iham**
fixes #26

### CODE REVIEW

**Comment by iham** [comment]
@jenkins-plone-org please run jobs

File: plone/app/contentmenu/tests/test_menu.py [hunk 1]
```diff
@@ -132,6 +132,7 @@ def testDefaultPageIncludesParentAndItemViewsWhenItemHasMultipleViews(self):  #
             'folderDefaultPageDisplay',
             [a['extra']['id'] for a in actions]
         )
+        import pdb; pdb.set_trace()
```
Comment (pysailor) [inline plone/app/contentmenu/tests/test_menu.py]: I don't think this is intended...

**Comment by iham** [comment]
@jenkins-plone-org please run jobs

**Comment by iham** [comment]
@pysailor can you sort of confirm merging?
i get only archetypes errors which i tend to ignore, as i don't think this is related to my changes.+

same (errors) goes for https://github.com/plone/Products.CMFDynamicViewFTI/pull/22

**Comment by iham** [comment]
@mauritsvanrees @jensens - maybe you could help on that?

File: plone/app/contentmenu/displayviewsmenu.py [hunk 2]
```diff
@@ -15,12 +18,20 @@ def getMenuItemByAction(self, context, request, action):
...
+
             item_action = item.action
             # Normalize menu item action; never uses ++view++
             if item_action.startswith('@@'):
                 item_action = item_action[2:]
 
             if item_action == action:
-                return item
+                permission = getUtility(IPermission, name=item.permission)
+                user_permitted = api.user.has_permission(
```
Comment (jensens) [inline plone/app/contentmenu/displayviewsmenu.py:32]: plone.api is not allowed in core in order to avoid circular dependencies.

**Review by jensens — DISMISSED** [review]

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

**Review by petschki — APPROVED** [review]
I've refactored the permission check. This now LGTM

**Review by mauritsvanrees — APPROVED** [review]

**Comment by petschki** [comment]
@mauritsvanrees yes saw that... zope permissions aren't strings but objects, so if we use `AccessControl` api here, it's safe for all cases ... should be green now

**Pull request merged**